docs/discussions.rst

```diff
@@ -115,9 +115,7 @@ their identifiers could be used (there's no hard limit on the length of a lock s
 but stay reasonable).
 
 A task can only take a single lock so there's no dead-lock scenario possible where two
-running tasks are waiting one another. That being said, if a worker dies with a lock, it
-will be up to you to free it. If the task fails but the worker survives though, the
-lock will be freed.
```

Comment:
It this outdated ?

Reply:
we could rephrasing it as "If a worker is killed without ending its job, following jobs with the same lock will not run until the interrupted job is either manually set to "failed" or "succeeded". If a job simply fails, following jobs with the same locks may run.